backend/api/Models/Models.cs

```diff
@@ -18,14 +19,85 @@ public class Robot
         [MaxLength(128)]
         [Required]
         public string SerialNumber { get; set; }
+        [MaxLength(128)]
+        [Required]
+        public string Logs { get; set; }
+        [MaxLength(128)]
+        [Required]
+        public string Host { get; set; }
+        [Required]
+        public int Port { get; set; }
+        [Required]
+        public bool Enabled { get; set; }
         [Required]
         public RobotStatus Status { get; set; }
     }
 
+    public class Report
```

Comment:
This should be in a separate file Report.cs

Reply:
Agreed